pkg/grpc/api.go

```diff
@@ -107,6 +115,22 @@ func (a *api) CallLocal(ctx context.Context, in *internalv1pb.InternalInvokeRequ
 		return nil, status.Errorf(codes.InvalidArgument, "parsing InternalInvokeRequest error: %s", err.Error())
 	}
 
+	operation := req.Message().Method
+	var httpVerb common.HTTPExtension_Verb
+
+	if req.Metadata() != nil && len(req.Metadata()) > 0 {
+		httpExt := req.Message().GetHttpExtension()
+		if httpExt != nil {
+			httpVerb = httpExt.GetVerb()
+		}
+	}
+
+	callAllowed := a.applyAccessControlPolicies(ctx, operation, httpVerb)
+
+	if !callAllowed {
+		return nil, status.Errorf(codes.PermissionDenied, "Access Control Policy has denied access: operation: %s verb: %s", operation, httpVerb)
```

Comment:
We want to make sure that this error messages appears in Debug logs. if the calling methods do not eventually log.Debug this, we probably want to do this here.

Reply:
added log